Utils/distroutils.py

```diff
@@ -10,16 +10,10 @@
 import Utils.constants as constants
 
 
-def get_my_distro(config):
+def get_my_distro(os_name, config):
```

Comment:
Overload this function instead of changing the signature. Other extensions might be making use of Utils.py, and they can't catch the changes at compile time because Python